trojsten/contests/models.py

```diff
@@ -339,6 +384,26 @@ class Task(models.Model):
     external_submit_link = models.CharField(
         max_length=128, verbose_name="Odkaz na externé odovzdávanie", blank=True, null=True
     )
+    text_submit_solution = ArrayField(
+        models.CharField(
+            max_length=512, verbose_name="textové riešenia (oddeľ čiarkou)", blank=True, null=True,
+        ),
+        blank=True,
+        null=True,
+        default=list,
+    )
+    susi_small_hint = ArrayField(
+        models.CharField(max_length=128, verbose_name="Suši malý hint", blank=True, null=True),
+        blank=True,
+        null=True,
+        default=list,
+    )
+    susi_big_hint = ArrayField(
+        models.CharField(max_length=128, verbose_name="Suši veľký hint", blank=True, null=True),
+        blank=True,
+        null=True,
+        default=list,
+    )
```

Comment:
hm... ja som povodne s tym ArrayFieldom myslel ze by sme mali pole hints, a prva polozka v nom by bol prvy hint a druha druhy...., taketo riesenie neviem ci neprinasa viac problemov ako uzitku oproti obycajnemu TextField-u.. (akoze ak cielom je mat viac hintov ktore sa zobrazia naraz, tak ok, len mne to zmysel nedava :smile: ak ucelom tohto arrayfieldu bol vyriesit problem co ked prekrocime limit znakov, tak na to je dlzkovo neobmedzeny TextField)

Reply:
No mne sa viac pozdava to, ze pri vytvarani ulohy su dve policka clearly urcene ako maly hint a velky hint (to mozno ide aj s ArrayFieldom, not sure). Nie kazda uloha bude mat oba, a podla toho aky silny je hint ho vieme priradit k malemu/velkemu a podla toho sa uberu zan body.
Asi by to islo dat aj do jedneho ArrayFieldu so size=2, ale neviem ci to nie je viac komplikovane ako keby dame dva CharFields. Asi mas ale pravdu, ze mat dva arrayfields je useless.
Este je otazka, ze ci sa da v CharFielde davat novy riadok.

Reply:
Da sa dat newline do CharFieldu aj ked defaultne ti to generuje iba jednoriadkove inputy... ale na hint by som radsej pouzil TextField... ma neobmedzenu dlzku a rovno defaultne ma textarea inputy